config/webpack/test.webpack.config.js

```diff
@@ -87,7 +76,16 @@ function getWebpackConfig(skyPagesConfig, argv) {
         {
           enforce: 'pre',
           loader: outPath('loader', 'sky-processor', 'preload'),
-          exclude: /node_modules/
+          exclude: excludes
+        },
+        {
+          test: /\.s?css$/,
+          use: ['raw-loader', 'sass-loader']
+        },
+        {
+          test: /\.html$/,
```

Comment:
Updated CSS, SCSS, and HTML rules to look just like common config (no affect on output).